crates/napi/package.json

```diff
@@ -41,17 +42,20 @@
   },
   "scripts": {
     "artifacts": "napi artifacts",
-    "build": "napi build --no-const-enum --platform --release",
+    "build": "napi build --no-const-enum --platform --release && yarn run typegen",
     "build:debug": "napi build --no-const-enum --platform",
     "prepublishOnly": "napi prepublish -t npm --skip-gh-release",
     "test": "ava",
-    "version": "napi version"
+    "version": "napi version",
+    "typegen": "tsx scripts/generate-types.ts"
```

Comment:
can we use ts-node? it can save several deps

Reply:
Oh, sorry, I didn't notice we already have `ts-node`